diff --git a/g2o/core/base_multi_edge.h b/g2o/core/base_multi_edge.h
index 68dbe2b..ba839a3 100644
--- a/g2o/core/base_multi_edge.h
+++ b/g2o/core/base_multi_edge.h
@@ -64,7 +64,7 @@ namespace g2o {
       typedef MatrixXD::MapType JacobianType;
       typedef typename BaseEdge<D,E>::ErrorVector ErrorVector;
       typedef typename BaseEdge<D,E>::InformationType InformationType;
-      typedef Eigen::Map<MatrixXD, MatrixXD::Flags & Eigen::PacketAccessBit ? Eigen::Aligned : Eigen::Unaligned > HessianBlockType;
+      typedef Eigen::Map<MatrixXD, Eigen::Aligned16 > HessianBlockType;
 
       BaseMultiEdge() : BaseEdge<D,E>()
       {
@@ -126,7 +126,7 @@ namespace g2o {
       typedef MatrixXD::MapType JacobianType;
       typedef typename BaseEdge<-1,E>::ErrorVector ErrorVector;
       typedef typename BaseEdge<-1,E>::InformationType InformationType;
-      typedef Eigen::Map<MatrixXD, MatrixXD::Flags & Eigen::PacketAccessBit ? Eigen::Aligned : Eigen::Unaligned > HessianBlockType;
+      typedef Eigen::Map<MatrixXD, Eigen::Aligned16 > HessianBlockType;
 
       BaseMultiEdge() : BaseEdge<-1,E>()
     {
